test/package/bundleCreate.test.ts

```diff
@@ -0,0 +1,176 @@
+/*
+ * Copyright (c) 2025, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import path from 'node:path';
+import fs from 'node:fs';
+import { expect } from 'chai';
+import { Connection, SfProject } from '@salesforce/core';
+import { instantiateContext, restoreContext, stubContext, MockTestOrgData } from '@salesforce/core/testSetup';
+import { BundleEntry } from '@salesforce/schemas';
+import { createBundle } from '../../src/package/packageBundleCreate';
+
+async function setupProject(setup: (project: SfProject) => void = () => {}) {
+  const project = await SfProject.resolve();
+
+  setup(project);
+  const projectDir = project.getPath();
+  project
+    .getSfProjectJson()
+    .getContents()
+    .packageDirectories?.forEach((dir) => {
+      if (dir.path) {
+        const packagePath = path.join(projectDir, dir.path);
+        fs.mkdirSync(packagePath, { recursive: true });
+      }
+    });
+
+  return project;
+}
+
+describe('bundleCreate', () => {
+  const testContext = instantiateContext();
+  const testOrg = new MockTestOrgData();
+  let connection: Connection;
+
+  beforeEach(async () => {
+    stubContext(testContext);
+    connection = await testOrg.getConnection();
+  });
+
+  afterEach(() => {
+    restoreContext(testContext);
+  });
+
+  describe('add bundle entry - no existing entries', () => {
+    it('add bundle entry - no existing entries', async () => {
+      testContext.inProject(true);
+      const project = await setupProject((proj) => {
+        proj.getSfProjectJson().set('namespace', 'testNamespace');
+      });
+      const bundleEntry: BundleEntry = {
+        name: 'testBundle',
+        versionName: 'testBundle',
+        versionNumber: '1.0.0',
+        versionDescription: 'testBundle',
+      };
+      project.getSfProjectJson().addPackageBundle(bundleEntry);
+      const bundles = (project.getSfProjectJson().getContents().packageBundles as BundleEntry[]) ?? [];
```

Comment:
? Didn't you add methods to get and work with packageBundles in core? use them - but I could be wrong